tms/clickup/serializer.py

```diff
@@ -29,11 +29,16 @@ class LoginSerializer(serializers.Serializer):
     username = serializers.CharField()
     password = serializers.CharField()
 
-    def create(self, validated_data):
-        username = validated_data["username"]
-        password = validated_data["password"]
-        user = authenticate(username=username, password=password)
-        if not user:
-            raise serializers.ValidationError(constant.LOGIN_ERROR_MESSAGE)
-        token, _ = Token.objects.get_or_create(user=user)
-        return token
\ No newline at end of file
+    def validate(self, data):
+        username = data.get('username')
+        password = data.get('password')
+
+        if username and password:
+            user = authenticate(username=username, password=password)
+            if not user:
+                raise serializers.ValidationError("Incorrect Username and Password. Please try again.")
+        else:
+            raise serializers.ValidationError("Both username and password are required.")
+
+        data['user'] = user
```

Comment:
don't need data[user] simply return return user
 def validate(self, data):
        user = authenticate(username=data['username'], password=data['password'])
        if not user:
            raise serializers.ValidationError({"detail": LOGIN_ERROR_MESSAGE})
        return user